aws-out-of-cluster.md

```diff
@@ -24,6 +24,9 @@ To allocate external AWS resources to a Kubernetes concept, use the following ta
 
 *\*In the `kubernetes_label_NAME` tag key, the `NAME` portion should appear exactly as the tag appears inside of Kubernetes. For example, for the tag `app.kubernetes.io/name`, this tag key would appear as `kubernetes_label_app.kubernetes.io/name`.*
 
+Additionally users must activate the `eks:cluster-name`
+![EKS cluster name user tag](images/user_eks_cluster_name_tag.png)
```

Comment:
Should we mention why? Or what happens if they don't? I'm torn on that, but it might be nice, to justify why they have to take this extra step.